src/features/sarcophagi/hooks/useGetSarcophagi.ts

```diff
@@ -21,11 +23,26 @@ export function useGetSarcophagi(sarcoIds: string[]): Sarcophagus[] {
     })),
   });
 
+  // Refetch the sarcohpagi on a set interval to update the sarcophagus status
+  useEffect(() => {
+    setSarcohpagiResponse(data as SarcophagusResponseContract[]);
+
+    const intervalId = setInterval(async () => {
+      const refetchedData = (await refetch?.()).data as SarcophagusResponseContract[];
```

Comment:
Would this throw an error if `refetch` is undefined, b/c `data` won't be available?